LGTM deprecation: Update basic queries to use VS Code (#11468)
github/codeql: docs/codeql/reusables/vs-code-basic-instructions/run-quick-query-1.rst

> The CodeQL extension for Visual Studio Code adds several **CodeQL:** commands to the command palette including **Quick Query**, which you can use to run a query without any set up.

Reviewer: Not sure about this one, you are really the experts here!
```suggestion
The CodeQL extension for Visual Studio Code adds several **CodeQL:** commands to the command palette including **Quick Query**, which you can use to run a query without any set-up.
```

Author: Thanks for checking. This is usually not hyphenated in US ENglish.

Reviewer 2: @felicitymay Isn't `setup` a single word when used as noun?

Author: I agree Arthur, I think this sentence is worded in such a way that it's not clear whether the term is intended to be a noun or a verb. I suggest we make the wording more explicit.

```suggestion
The CodeQL extension for Visual Studio Code adds several **CodeQL:** commands to the command palette including **Quick Query**, which you can use to create and run a query without setting up a CodeQL pack.
```

#. From the command palette in Visual Studio Code, select **CodeQL: Quick Query**.

> #. After a momment, a new tab *quick-query.ql* is opened, ready for you to write a query for your currently selected CodeQL database (here a |language-code| database).

Reviewer: ```suggestion
#. After a moment, a new tab *quick-query.ql* is opened, ready for you to write a query for your currently selected CodeQL database (here a |language-code| database).
```

Author: ```suggestion
#. After a moment, a new tab *quick-query.ql* is opened, ready for you to write a query for your currently selected CodeQL database (here a |language-code| database). If you are prompted to reload your workspace as a multi-folder workspace to allow Quick queries, accept or create a new workspace using the starter workflow.
```

> #. After a momment, a new tab *quick-query.ql* is opened, ready for you to write a query for your currently selected CodeQL database (here a |language-code| database).

Reviewer: Just doing this on the random workspace I had open, I got the message
```
Quick query requires multiple folders in the workspace. Reload workspace as multi-folder workspace?
```
I guess this does not happen on the starter workspace, but we might want to warn people and perhaps ensure them that reloading does not mess up their workspace?

Reviewer 3: Good point. I guess we could add an additional step saying something like 'If prompted, click the notification to open a multi-folder workspace'?

Reviewer: Yes that might be enough to reassure users (especially first-time users) that they are still on track.

Author: Thanks, I've amended @yoff's suggestion above with the spelling fix.
